apps/centralised-indexer/src/models/dftx/Token.ts

```diff
@@ -0,0 +1,156 @@
+import { Schema } from 'dynamoose'
+import { Injectable } from '@nestjs/common'
+import { InjectModel, Model } from 'nestjs-dynamoose'
+import BigNumber from 'bignumber.js'
+import { SortOrder } from 'dynamoose/dist/General'
+
+const FIXED_PARTITION_KEY = 0
+const DCT_ID_START: number = 128
+
+export interface TokenKey {
+  id: number
+}
+
+export interface Token extends TokenKey {
+  symbol: string
+  name: string
+  decimal: number
+  limit: string
+  isDAT: boolean
+  isLPS: boolean
+  tradeable: boolean
+  mintable: boolean
+
+  block: {
+    hash: string
+    height: number
+    time: number
+    medianTime: number
+  }
+
+  _fixedPartitionKey?: 0
+}
+
+export const TokenSchema = new Schema({
+  id: {
+    type: Number,
+    hashKey: true
+  },
+  symbol: String,
+  name: String,
+  decimal: Number,
+  limit: String,
+  isDAT: Boolean,
+  isLPS: Boolean,
```

Comment:
```suggestion
  isLPS: Boolean,
  isLoanToken: Boolean,
```
i think we need `isLoanToken` property as well